Assignment/solution_student/aboutme.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<meta charset="UTF-8">
+	<title>About Me</title>
+	<link rel="stylesheet" href="css/style.css">
+</head>
+<body>
+
+	<nav> 
+		<a href="aboutme.html">About Me</a> |
+		<a href="portfolio.html">Portfolio</a>
+	</nav>
+
+	<h1><center>JOE JOHNSON</center></h1>
```

Comment:
We don't want to use presentational markup like `<center>` at all, since it's outdated and only used to add style, which we want to leave to our CSS. You can do this with `text-align: center;`